Userland/Libraries/LibGL/GLTexture.cpp

```diff
@@ -4,8 +4,8 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include <LibGL/GL/gl.h>
 #include "GLContext.h"
+#include <LibGL/GL/gl.h>
```

Comment:
Clang-format did this? W e i r d 

(The fixed position is correct, what it was at...not so much)

Reply:
I might've run `clang-format` on this in the wrong commit... Whoops... I'll have to comb through and get that fixed..